subprojects/model-core/src/main/java/org/gradle/internal/reflect/validation/TypeValidationProblemRenderer.java

```diff
@@ -34,25 +34,51 @@ public static String renderMinimalInformationAbout(TypeValidationProblem problem
     }
 
     public static String renderMinimalInformationAbout(TypeValidationProblem problem, boolean renderDocLink) {
```

Comment:
I think we need unit tests for `TypeValidationProblemRenderer`, that check the actual output for a type validation problem. This allows us to check if problems are rendered as expected.

Reply:
What about testing `ValidationMessageChecker` instead? I don't think there's much value in testing this renderer in particular. However the fixture would be nice to test because it's what is actually displayed to the user and would be the actual messages we show to users.